### PR TITLE
Basic structure and responsive design

### PR DESCRIPTION
**Pull request by IevaRugine**

### CODE REVIEW

**Review by Andzej-F — DISMISSED** [review]

**Review by Slashass — DISMISSED** [review]

File: index.html [hunk 1]
```diff
@@ -27,70 +27,185 @@
...
+            </div>
         </div>
     </div>
-    <!-- watch video -->
-    <div class="container-fluid">
+    <!-- pricing -->
+    <div class="container">
         <div class="row">
-            WATCH VIDEO
+            <div class="col-12">
```
Comment (Katerynukr) [inline index.html:123]: we dont have price content

**Review by Katerynukr — CHANGES_REQUESTED** [review]
the structure should be changed according to our project. This changed are done only based on class project

File: css/layout.css [hunk 2]
```diff
@@ -5,36 +5,299 @@ body {
 
 .container-fluid {
     display: grid;
-    grid-template-columns: 1fr;
+    grid-template-columns: 15px 1fr 15px;
```
Comment (IevaRugine) [inline css/layout.css:8]: It is not 1 fr - we have gap's on each side of page

File: css/layout.css [hunk 3]
```diff
@@ -5,36 +5,299 @@ body {
...
+    padding: 20px 0;
+    border: 1px solid #ccc;
 }
-    .container-fluid > .row {
-        grid-column: 1 / 2;
-    }
```
Comment (IevaRugine) [inline css/layout.css:12]: ???

**Review by Andzej-F — APPROVED** [review]

File: index.html [hunk 4]
```diff
@@ -22,40 +22,32 @@
...
-                <div class="filter">
-                    FILTER
-                </div>
-                <div class="list">
-                    LIST
-                </div>
+            <div class="col-6">
+                IMG
+            </div>
+            <div class="col-12 col-md-12">
```
Comment (Katerynukr) [inline index.html:46]: should have as well class col-6

File: index.html [hunk 5]
```diff
@@ -154,20 +160,14 @@ <h2>Blog</h2>
...
-        </div>
-        <div class="row">
-            <div class="col-12">
-                TESTIMONIALS PAGINATION
+                <h2>Sponsors logo</h2>
             </div>
         </div>
     </div>
-    </div>
     <!-- contact us -->
```
Comment (Katerynukr) [inline index.html:171]: we dont have this part at all

File: index.html [hunk 6]
```diff
@@ -184,27 +184,21 @@ <h2>Get in touch</h2>
...
             </div>
         </div>
         <div class="row">
             <div class="col-12">
-                SOCIALS
+                Newsletter
             </div>
         </div>
         <div class="row">
             <div class="col-12">
```
Comment (Katerynukr) [inline index.html:200]: col-2

File: index.html [hunk 7]
```diff
@@ -184,27 +184,21 @@ <h2>Get in touch</h2>
...
     <!-- footer -->
     <footer class="container">
         <div class="row">
             <div class="col-12">
-                LOGO
+                About Me
             </div>
         </div>
         <div class="row">
             <div class="col-12">
```
Comment (Katerynukr) [inline index.html:195]: col-5

File: index.html [hunk 8]
```diff
@@ -184,27 +184,21 @@ <h2>Get in touch</h2>
...
-    <!-- maps -->
-    <div class="container-fluid">
-        <div class="row">
-            MAP
-        </div>
-    </div>
     <!-- footer -->
     <footer class="container">
         <div class="row">
             <div class="col-12">
```
Comment (Katerynukr) [inline index.html:190]: this should be col-5

**Review by AlgisP — APPROVED** [review]

**Comment by Katerynukr** [comment]
please, lets make a more organized file basis ... it is gonna be easier to work on if it is properly done

**Review by IevaRugine — COMMENTED** [review]
> please, lets make a more organized file basis ... it is gonna be easier to work on if it is properly done

Of course we can, tomorrow after lecture.  Have a good evening! :)

**Pull request merged**